src/templates/styles/locations.scss

```diff
@@ -59,7 +59,7 @@
       display: flex;
       flex-flow: row wrap;
       margin: 0;
-      padding: 0;
+      padding-bottom: .5rem;
```

Comment:
before
```
padding: 0
=> top: 0, right: 0, bottom: 0, left: 0
```

after:

```
padding-bottom: .5rem;
top: [default of ul], right: [default of ul], bottom: 0, left: [default of ul]
```

Reply:
Thanks, when I had dev tools open and made the change I didn't notice that default padding-left of ul.